index.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
         'jest/no-focused-tests': 'error',
         'jest/no-identical-title': 'error',
         'jest/no-jest-import': 'error',
+        'jest/no-mocks-import': 'error',
```

Comment:
I think it's always an error so it belongs in recommended? For linting I'd even say just release new default rules when they're ready, it's better to have many small majors than one big major where people see hundreds of errors after upgrading and decide to just disable the new rules because they're annoying ^^

Reply:
It's still a breaking change. And it might not always be an error even though I agree it's not a good idea

Reply:
So to be clear, are you suggesting not to add it to recommended or to add it to recommended at a later point?

Reply:
Add it to recommended at a later point :)

Reply:
Okay. We can leave the line adding it to recommended commented out to keep track. Are there any other rules that you'd want to go into recommended in the next major?

Reply:
Yeah, a comment is a good idea.

I'd like to activate the ones for using circus at least

Reply:
I changed it. Didn't add a commented out one for jasmine globals because it's already there (just as `warn`)